source/fleetprovisioning/README.md

```diff
@@ -216,11 +216,13 @@ To get started with the feature you will need to set the right configuration. Th
 
 `device-key`: Path to the device private key.
 
+`template-parameters`: The Fleet Provisioning Template parameters. A JSON object specified as an escaped string.
```

Comment:
I think it would be highly beneficial to provide a concrete example of what someone can pass in here. IE if I want to pass a serial number parameter to the fleet provisioning feature, what should my input here look like?

Reply:
I think it would also be helpful to point the reader to any available resources that might show them how to create or modify an existing FP policy/template that can process the template parameters being passed in.